db/migrate/20191122225846_make_verifying_agencies.rb

```diff
@@ -0,0 +1,13 @@
+class MakeVerifyingAgencies < ActiveRecord::Migration[5.2]
+  def up
+    %w[UHS Disability\ Services Accessible\ Workplace].each do |name|
```

Comment:
When more than half of the "words" have spaces in them, I prefer a regular array literal. This way only saves you 3 characters. And, because the `\`es look like dividers, I parsed this as, "UHS Disability, Services Accessible, Workplace," on the first pass.